Python/BinarySearch.py

```diff
@@ -0,0 +1,18 @@
+def binarySearch(arr, left, right, num):
+    while (left <= right):
```

Comment:
no need to use parenthesis in python. They are optional, and only used for very complex conditions.